pkg/domain/filters/volumes.go

```diff
@@ -61,6 +61,29 @@ func GenerateVolumeFilters(filters map[string][]string) ([]libpod.VolumeFilter,
 					}
 					return false
 				})
+			case "dangling":
+				danglingVal := val
+				invert := false
+				switch strings.ToLower(danglingVal) {
+				case "true":
```

Comment:
Should this be case "true", "1":

And the other be case "false", "0"

Reply:
Fixed